nix/haskell.nix

```diff
@@ -40,6 +40,8 @@ let
       };
     modules = [
         {
+          compiler.nix-name = pkgs.lib.mkForce "ghc8102-experimental";
```

Comment:
haskell.nix nit: it would be nice if you could pass this as an argument to `stackProject` to make it maximally similar to `cabalProject`!